src/Native/Unix/System.Native/pal_io.h

```diff
@@ -329,98 +329,104 @@ enum NotifyEvents
     PAL_IN_ISDIR = 0x40000000,
 };
 
+DLLEXPORT int32_t SystemNative_Stat2(const char* path, struct FileStatus* output);
```

Comment:
You can just rename `SystemNative_Stat` to `SystemNative_Stat2` below. `SystemNative_Stat` does not exist at this moment.